stats/lib/platform-stats-fetchers.js

```diff
@@ -136,22 +136,27 @@ export const fetchTopEarningParticipants = async (pgPool, filter) => {
   assert(filter.to === today(), 400, 'filter.to must be today, other values are not supported')
   const { rows } = await pgPool.query(`
     WITH latest_scheduled_rewards AS (
-      SELECT DISTINCT ON (participant_address) participant_address, scheduled_rewards
-      FROM daily_scheduled_rewards
-      ORDER BY participant_address, day DESC
+      SELECT dsr.participant_id, dsr.scheduled_rewards
+      FROM daily_scheduled_rewards dsr
+      WHERE dsr.day = (
+        SELECT MAX(day) 
+        FROM daily_scheduled_rewards dsr2 
+        WHERE dsr2.participant_id = dsr.participant_id
+      )
```

Comment:
I don't understand why we need this more complex query. Wouldn't the following work?

```sql
      SELECT DISTINCT ON (participant_id) participant_id, scheduled_rewards
      FROM daily_scheduled_rewards
      ORDER BY participant_id, day DESC
```